Frontend/src/Components/Graph/ExpandableContent.js

```diff
@@ -0,0 +1,32 @@
+import { useState } from "react";
+import { Collapse, Box, Button } from "@chakra-ui/react";
+import { ChevronDownIcon } from "@chakra-ui/icons";
+
+function Expandable(props) {
+  const [isExpanded, setIsExpanded] = useState(false);
+
+  return (
+    <Box maxW="2xl">
+      <Box>
+        <Button
+          onClick={() => setIsExpanded(!isExpanded)}
+          leftIcon={<ChevronDownIcon />}
+          variant="ghost"
+          colorScheme="gray"
+          size={props.size}
+        >
+          {props.label}
+        </Button>
+      </Box>
+      <Collapse in={isExpanded} animateOpacity>
+        <Box p="3">
+          <Box p="4" bg="gray.50" rounded="md" shadow="md">
+            {props.children}
+          </Box>
+        </Box>
```

Comment:
```suggestion
        <Box m="3" p="4" bg={props.contentBg} rounded="md" shadow="md">
          {props.children}
        </Box>
```
1. Rather than having two Boxes, we can just use one Box with an additional `m` (margin) of 3.
2. The static `bg` value doesn't work with changing color modes (see image below). To fix this, you could add a `contentBg` color in `colors.js` (in the `Components/Shared` directory) for light and dark modes (going off of [Chakra UI's default theme](https://chakra-ui.com/docs/styled-system/theme), `#f7fafc` and `#4a5568` could work well). Then, you can pass this as a prop to the `Expandable` component.
   1. Note - I would put the call to `getColor()` in `CustomGraph.js` or `GraphSelectModal.js` and then pass it down to the Expandable component as a prop so that `getColor()` isn't being evaluated for every Expandable instance.

Modal in dark mode with static `bg` value:
![image](https://user-images.githubusercontent.com/72111610/235374788-f09bf14c-2937-4160-a46b-8ac89a951008.png)

Reply:
fixed